modules/platen/layouts/partials/platen/markup/subscript/image.html

```diff
@@ -0,0 +1,35 @@
+{{- $Params       := .                                                         -}}
+{{- $Page         := $Params.Page                                              -}}
+{{- $destination  := $Params.Destination                                     -}}
+{{- $title        := $Params.Title                                             -}}
+{{- $text         := $Params.Text                                              -}}
+{{- $IsBlock      := $Params.IsBlock                                           -}}
+{{- $Ordinal      := $Params.Ordinal                                            -}}
+{{- $ConfigKey   := "platen.markup.subscript"                                        -}}
+{{- $Config      := partialCached "platen/param/getKey" $ConfigKey $ConfigKey -}}
+{{- $Aliases     := $Config.aliases                                                  -}}
```

Comment:
Nit: alignment for code style
```suggestion
{{- $Params      := .                                                         -}}
{{- $Page        := $Params.Page                                              -}}
{{- $destination := $Params.Destination                                       -}}
{{- $title       := $Params.Title                                             -}}
{{- $text        := $Params.Text                                              -}}
{{- $IsBlock     := $Params.IsBlock                                           -}}
{{- $Ordinal     := $Params.Ordinal                                           -}}
{{- $ConfigKey   := "platen.markup.subscript"                                 -}}
{{- $Config      := partialCached "platen/param/getKey" $ConfigKey $ConfigKey -}}
{{- $Aliases     := $Config.aliases                                           -}}
```